### ISSUE TITLE
Library: Pull to Refresh

### ISSUE THREAD

**Issue by luke-freeman** [razeware/emitron-iOS#489]
Not really an issue but it would be nice to have some kind of pull to refresh

**Comment by byaruhaf** [comment]
Since the Pull-to-refresh UI/UX pattern is used to request a data update in most apps, wouldn't this be more useful for an app whose backend data is regularly updated?

**Comment by 0xTim** [comment]
(It's also not easily possible with SwiftUI until iOS 15)

**Comment by VegetarianZombie** [comment]
@0xTim how do you feel about us adding support to this feature for iOS 15 and leaving as-is for iOS 14?

**Comment by 0xTim** [comment]
@VegetarianZombie Yep agree since it's added natively on iOS 15 rather than trying to hack it in
